### PR TITLE
Add Vungle examples

### PR DESCRIPTION
**Pull request by victor-takai**

### CODE REVIEW

File: nimbus-ios-sample/Ad Manager/AdDemoViewController.swift [hunk 1]
```diff
@@ -8,30 +8,8 @@
 import UIKit
 import NimbusKit
 
-#if canImport(NimbusSDK)
```
Comment (inderdhir) [inline nimbus-ios-sample/Ad Manager/AdDemoViewController.swift:11]: Can we confirm both targets (pods and SPM) AND the internal sample build and run successfully with these removed?

File: nimbus-ios-sample/Common/DemoCell.swift [hunk 2]
```diff
@@ -52,23 +52,17 @@ class DemoCell: UITableViewCell {
...
-        label.accessibilityIdentifier = adType.getIdentifier(prefix: "adDemo", .cellLabel)
-    }
-    
     func updateWithMediationAdType(_ adType: MediationAdType) {
         accessibilityIdentifier = adType.getIdentifier(prefix: "mediationPlatforms", .cell)
         label.text = adType.description
         label.accessibilityIdentifier = adType.getIdentifier(prefix: "mediationPlatforms", .cellLabel)
     }
+    
+    func updateWithThirdPartyDemanAdType(_ adType: ThirdPartyDemandAdType) {
```
Comment (inderdhir) [inline nimbus-ios-sample/Common/DemoCell.swift:61]: Typo: `updateWithThirdPartyDemandAdType`

File: nimbus-ios-sample/Third Party Demand/ThirdPartyDemandAdType.swift [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        case .apsInterstitialHybrid:
+            return "APS Interstitial Hybrid"
+        case .vungleMREC:
+            return "Vungle MREC"
+        default:
+            return rawValue.camelCaseToWords()
+        }
+    }
+    
+    public static var apsAdType = [apsRefreshingBanner, apsInterstitialHybrid]
```
Comment (inderdhir) [inline nimbus-ios-sample/Third Party Demand/ThirdPartyDemandAdType.swift:27]: rename these 4 vars by replacing `Type` to `Types` as it's an array

File: nimbus-ios-sample/Third Party Demand/Vungle/VungleViewController.swift [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+                format: .letterbox
+            )
+            
+        case .vungleInterstitial:
+            return NimbusRequest.forInterstitialAd(
+                position: ConfigManager.shared.vungleInterstitialPlacementId!
+            )
+            
+        case .vungleRewarded:
+            return customNimbusRequestForRewardedVideo(
```
Comment (inderdhir) [inline nimbus-ios-sample/Third Party Demand/Vungle/VungleViewController.swift:134]: Use `NimbusRequest.forRewardedVideo` here instead

**Review by inderdhir — APPROVED** [review]
Left a few minor comments

File: nimbus-ios-sample/Third Party Demand/Vungle/VungleViewController.swift (hunk 4 above)
Comment (victor-takai) [reply nimbus-ios-sample/Third Party Demand/Vungle/VungleViewController.swift:134]: This method already calls `NimbusRequest.forRewardedVideo`

**Review by brugg — APPROVED** [review]

**Pull request merged**